static/app/views/releases/drawer/releasesDrawerList.tsx

```diff
@@ -67,6 +67,23 @@ export function ReleasesDrawerList({
               </Fragment>
             }
             Visualization={chartRenderer?.({
+              chartRef: (e: ReactEchartsRef | null) => {
+                if (e) {
+                  // When chart is mounted, zoom the chart into the relevant
+                  // bucket
+                  e.getEchartsInstance().dispatchAction({
+                    type: 'dataZoom',
+                    batch: [
+                      {
+                        // data value at starting location
+                        startValue: startTs,
+                        // data value at ending location
+                        endValue: endTs,
+                      },
+                    ],
+                  });
```

Comment:
Sooo in `TimeSeriesWidgetsViz` we trim the series that the chart renders to be within the release bucket timeframe, but this could be an alternative where we have the chart just "zoom" into the proper timeframe.